docs/cases/cases-overview.asciidoc

```diff
@@ -31,7 +31,11 @@ image::images/cases-home-page.png[Case UI Home]
 Open a new case to keep track of security issues and share their details with
 colleagues.
 
-. Go to *Cases* -> *Create new case*.
+. Go to *Detect* -> *Cases*, then click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table. 
```

Comment:
```suggestion
. Go to *Detect* -> *Cases*, then click *Create case*. If no cases exist, you can create one by clicking the *Create case* button inside the table. 
```
I think we probably don't need to tell them the cases table will be empty since they're sure to see that. In fact maybe we don't even need the second sentence in this line since whether or not there are pre-existing cases, they can still click the same Create case button.

Reply:
+1 for Ben's suggestion.

Reply:
Yeah, I debated on describing the table's empty state. I agree that it doesn't add much value since, like you said, users can click on the button above the table regardless of whether cases exist. The button name in the empty table is the same too, so although the workflow to access the button is different, users are essentially clicking on the same button. IMO, the screenshot isn't completely necessary but it wouldn't hurt to keep either. WDYT @benironside @jmikell821 ?

Reply:
I think cases are under the `Investigate` section and not under the `Detect`

<img width="190" alt="Screenshot 2022-02-01 at 12 32 25 PM" src="https://user-images.githubusercontent.com/7871006/151953036-aa18bf5c-e479-4102-bb7d-e1b62d8e9a98.png">

Reply:
That they are, good catch - thanks! I'll make this change right now.